### PR TITLE
ndx-sound

### PR DESCRIPTION
**Pull request by luiztauffer**
the `README.md` for this extension contains images hosted in the original repository, should we include images to `staged-extensions`?
@rly

### CODE REVIEW

**Comment by luiztauffer** [comment]
fyi @bendichter

File: ndx_sound/README.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+### Visualization
+
+Use `plot_sound` on its own outside nwbwidgets.
+```python
+from ndx_sound.widgets import plot_sound
+
+plot_sound(nwbfile.stimulus["acoustic_stimulus"])
+```
+
+![](ndx_sound_plot.png)
```
Comment (rly) [inline ndx_sound/README.md:49]:
```suggestion
![](https://github.com/catalystneuro/ndx-sound/blob/main/ndx_sound_plot.png)
```
I think you can link to the image on the main repo.

File: ndx_sound/README.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Use `acoustic_waveform_widget` to include an Audio element that plays the sound.
+
+```python
+from ndx_sound.widgets import acoustic_waveform_widget
+
+acoustic_waveform_widget(nwb.stimulus["acoustic_stimulus"])
+```
+
+![](acoustic_waveform_widget.png)
```
Comment (rly) [inline ndx_sound/README.md:59]:
```suggestion
![](https://github.com/catalystneuro/ndx-sound/blob/main/acoustic_waveform_widget.png)
```

**Review by rly — APPROVED** [review]

**Comment by rly** [comment]
Looks good to me! I will hold off on merging the PR and adding to he catalog until #33 is resolved.

**Comment by weiglszonja** [comment]
@rly @luiztauffer I updated `ndx_sound/ndx-meta.yaml` and `ndx_sound/README.md` with the new version of [ndx-sound]( https://pypi.org/project/ndx-sound/).

**Pull request merged**